### PR TITLE
Update and interpolate VMT data, update test column names

### LINKED ISSUE #111: Update and interpolate VMT data

### PR DESCRIPTION
**Pull request by eroten**
IMPORTANT: Quarto does not successfully render due to changes in the column names across sectors. I ensured the transportation sections all run

- Vehicle miles traveled
  - MN and WI DOT VMT now extend backwards to the earliest published data (2001 for MN, 2009 for WI).
  - MnDOT now has 2023 VMT
  - For scaling statewide emissions to the county level using VMT, we need a complete time series from our base year (2005) to current year.  MnDOT VMT for year 2015 is unavailable due to _significant and fundamental_ changes in data structure that make directly comparing data prior and post 2015 inappropriate. However, our interpolation here is based on the county level summary of all VMT and centerline mileage using the midpoint method. The midpoint method takes the average of the existing observations directly before and after the missing observation in the sequence. Essentially, we are drawing a direct line between 2014 and 2016. Our interpolation is also only for a single year.
  - Thus, MnDOT daily, annual VMT and centerline mileage has been interpolated for year 2015 using the midpoint method (`zoo::na.approx()`), after summarizing at the county and year levels. Route system was dropped due to changes in route system classifications from year to year. 
  - `dot_vmt` has new column, `cprg_area`, which indicates if the county is in the CPRG study area. This makes for easier joining later on, particularly because there is a Washington County in both MN and WI, but only the MN county is included in the study area. 
  - Wisconsin VMT data processing is now completed with {tabulapdf}

Testing changes
- Tests for dot_vmt updated
- Double checked tests with new column names across all sectors

Quarto changes
- Ensured all dependencies for `dot_vmt` render correctly after adding additional data.
- Ensured all transportation Quarto documents render with new column names
- VMT data documentation sections updated

Closes #111 

## Checklist

Please complete this checklist as a courtesy to the PR reviewer.  

Code and styling
* [x] All of the files/scripts I added are in the right place and named appropriately. See the [README](https://github.com/Metropolitan-Council/ghg-cprg/blob/main/README.md) for details.
* [x] I have not used `setwd()`
* [x] I have used `file.path(here::here(), "file_name"))` to source any scripts or read in data
* [x] I have not added any large datasets, unless absolutely necessary (explain)
* [x] I have commented my code, particularly in hard to understand areas
* [x] I have added additional package dependencies as necessary with `renv::install()`
* [x] I have run `styler::style_dir(".", recursive = TRUE, filetype = c("R", "qmd"))`
* [x] Plots
  - [x] If plotly, use `source = "chunk-name"` in `plot_ly()`
  - [x] Use formatting with `councilR::plotly_layout()`?
* [x] Chunk formatting
  - [x] All chunks named
  - [x] All figure or table chunks have caption
  - [x] `out.width: "95%"`. If a specific height is needed, use pixels. and/or `out.height: "500px"`

Document editing
* [ ] I have ensured that modified documents knit successfully from [`render_for_publication.R`](https://github.com/Metropolitan-Council/ghg-cprg/blob/main/R/render_for_publication.R) SEE NOTE ABOVE
* [x] I have fixed any missing citations, cross references, hyperlinks
* [x] I have reviewed my contributions for typos and misspellings.

GitHub and project management
* [x] I have identified and assigned at least one Reviewer (Liz, Sam, Laine) to this PR
* [x] I have assigned myself to this PR
* [x] I have updated the status in the [GitHub Project](https://github.com/orgs/Metropolitan-Council/projects/8/views/1)

### CODE REVIEW

File: _agriculture/data-raw/livestock-emissions-2021.R
Comment (pawilfahrt) [inline _agriculture/data-raw/livestock-emissions-2021.R:1]: I'm ignoring all agriculture files here as they have all been substantially revised on agriculture-emissions-2005 branch. Plus it looks like styler changes at a glance

File: _transportation/data-raw/mndot_vmt_county_functional_class.R
Comment (pawilfahrt) [inline _transportation/data-raw/mndot_vmt_county_functional_class.R:1]: @eroten  can you add all of the crs files from 2001 onward to the teams file so I can download from there directly?

Comment (eroten) [reply _transportation/data-raw/mndot_vmt_county_functional_class.R:1]: Done!

File: _transportation/data_state_dot.qmd
Comment (pawilfahrt) [inline _transportation/data_state_dot.qmd:1]: Can't render as wi_osm_meta is not found in current environment, but assuming this is in broader transportation branch so I think it's okay for now.

**Review by pawilfahrt — APPROVED** [review]
Everything's looking good! Most of my edits were updating naming conventions in scripts.

**Pull request merged**